modules/serverless-eventpolicy-resource-overview.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-event-authorization-eventpolicy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-eventpolicy-resource-overview_{context}"]
+= EventPolicy resource overview
+
+The EventPolicy resource defines rules for event delivery. It specifies which entities like service accounts or event sources are allowed to send events, which consumers may receive events, and optionally, additional criteria that CloudEvents must match to be accepted.
+
+An EventPolicy is structured with three primary sections as follows:
+
+[cols=2,options]
+|===
+
+|`.spec.to`
+|Defines the resources such as Brokers or Channels that the EventPolicy applies to.
+
+|`.spec.from`
+|Defines which sources or subjects are authorized to send events to the targets.
+
+|`.spec.filters`
+|Specifies advanced filtering conditions that must be satisfied by the CloudEvent itself before it is allowed.
```

Comment:
Maybe add an hint, that this field is optional

Reply:
Done